web/public/gameData_v1.0-23.json

```diff
@@ -10221,7 +10633,377 @@
                 "power": 4
             },
             "isAlternate": false,
-            "isFicsmas": false
+            "isFicsmas": false,
+            "isPowerGenerator": false
+        }
+    ],
+    "powerGenerationRecipes": [
+        {
+            "id": "GeneratorBiomass_Automated_GenericBiomass",
+            "displayName": "Biomass Burner (Biomass)",
+            "ingredients": [
+                {
+                    "part": "GenericBiomass",
+                    "amount": 0.16666666666666666,
+                    "perMin": 10
+                }
+            ],
+            "products": [],
+            "building": {
+                "name": "BiomassBurner",
+                "power": 30
```

Comment:
So is this where the power generated is defined?

Because I know Geo Thermal power operates on a scale, so I'm not sure if this would work like this. Are there any other types of power that operate on a min / max?

Reply:
there are none. We should treat it the same way we do particle accelerators and quantum encoders. I'll add the missing min/max.